src/ports/postgres/modules/utilities/utilities.py_in

```diff
@@ -709,16 +709,35 @@ def _check_groups(tbl1, tbl2, grp_list):
     return ' AND '.join([" {tbl1}.{i} = {tbl2}.{i} ".format(**locals())
                          for i in grp_list])
 
-
-def _grp_from_table(tbl, grp_list):
-    """
-    Helper function for selecting grouping columns of a table
+def get_filtered_cols_subquery_str(include_from_table, exclude_from_table,
```

Comment:
what is our consideration to define function name not start with underscore(weak internal use) while the previous one is a weak internal use?

Reply:
we removed the _ because get_filtered_cols_subquery_str is not for internal use, it is called by pagerank and the python files.